hw4 - js-classes/inheritance-es5.js

```diff
@@ -0,0 +1,110 @@
+function Car(name, model, year, color, maxSpeed, fuelCapacity = 60, fuelConsumption = 10) {
+    this.name = name;
+    this.model = model;
+    this.year = year;
+    this.color = color;
+    this.maxSpeed = maxSpeed;
+    this.fuelCapacity = fuelCapacity;
+    this.fuelConsumption = fuelConsumption;
+}
+
+const newCar = new Car ('Lamborghini', 'Aventador Roadster', 2015, 'Yellow & Black', 350);
+
+Car.prototype.getFullName = function() {
+    return this.name + ' ' + this.model;
```

Comment:
It's better to use ES6 syntax:
```suggestion
    return `${this.name} ${this.model}`;
```